.github/actions/start-opensearch-with-one-plugin/action.yml

```diff
@@ -0,0 +1,126 @@
+name: 'Start OpenSearch with One Plugin'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
+    required: true
+
+  plugin-start-script:
+    description: 'The file name for the configuration script for the plugin such as install_demo_configurations -- may not be needed for every plugin'
+    required: false
+
+  plugin-zip:
+    description: 'The name of the zip file for the plugin hosted on docker-host i.e. security-plugin.zip '
```

Comment:
This may be confusing for someone who is unaware that this actions uses docker to start the node under the hood. What do you think of `A zip of the build of the plugin` or something similar?

Reply:
The issue with swapping to any other naming convention is that it becomes further from the actual case. Saying a zip of the build of the plugin seems to suggest that it is just a zip but this is strictly for the file hosted on docker-host. So I think until we can rename the docker host file it is going to have to stay as it is.